lunavl/sdk/luna_faces.py

```diff
@@ -324,6 +327,40 @@ def detect(self, images: List[Union[VLImage, ImageForDetection]], limit: int = 5
             )
         return res
 
+    def redetectOne(self, image: ImageForRedetections) -> List[VLFaceDetection]:
```

Comment:
redetectOne return <= 1 Face

Reply:
no! it returns a detection or raises an exception